themes/solr/templates/operator/downloads.html

```diff
@@ -0,0 +1,160 @@
+{% extends "operator/page.html" %}
+
+{% block content_inner %}
+<div class="small-12 columns">
+  <style type="text/css">
+    .headerlink, .elementid-permalink {
+      visibility: hidden;
+    }
+    h2:hover > .headerlink, h3:hover > .headerlink, h1:hover > .headerlink, h6:hover > .headerlink, h4:hover > .headerlink, h5:hover > .headerlink, dt:hover > .elementid-permalink {
+      visibility: visible
+    }
+  </style>
+
+  <h1 id="solr-downloads">{{ page.title }}
+    <a class="headerlink" href="#solr-downloads" title="Permanent link">¶</a>
+  </h1>
+  {{ page.content }}
+  <h2 id="solr-{{ SOLR_OPERATOR_LATEST_RELEASE | replace(".", "") }}">Solr Operator {{ SOLR_OPERATOR_LATEST_RELEASE }}
+    <a class="headerlink" href="#solr-{{ SOLR_OPERATOR_LATEST_RELEASE | replace(".", "") }}" title="Permanent link">¶</a>
+  </h2>
+
+  <p>Solr Operator {{ SOLR_OPERATOR_LATEST_RELEASE }} is the most recent Apache Solr Operator release.</p>
+
+  <ul>
+    <li>Source release:
+      <a href="https://www.apache.org/dyn/closer.lua/solr/operator/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz">solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz</a>
+      [<a href="https://downloads.apache.org/solr/operator/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz.asc">PGP</a>]
+      [<a href="https://downloads.apache.org/solr/operator/{{ SOLR_OPERATOR_LATEST_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_LATEST_RELEASE }}.tgz.sha512">SHA512</a>]
+    </li>
+
+    <li>Docker releases:
+      <a href="https://hub.docker.com/r/apache/solr-operator">apache/solr-operator</a>
+    </li>
+
+    <li>Helm Chart release:
+      <a href="https://artifacthub.io/packages/helm/apache-solr/solr-operator/{{ SOLR_OPERATOR_LATEST_RELEASE  | replace("v", "")  }}">Artifact Hub</a>
+    </li>
+
+    <li>CRDs:
+      <ul>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_LATEST_RELEASE }}/all.yaml">All</a></li>
```

Comment:
Should the CRDs live on the website, or will it be in a location "softlinked" to somewhere else?

Reply:
It's redirected to the same place as the helm charts, nightlies.apache.org (hopefully they will have a better location in the future.) The files do not live in the site.

Reply:
https://github.com/apache/solr-site/blob/main/themes/solr/templates/htaccess.template#L85